src/Traits/FlowIdentifierTrait.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+
+namespace Isobar\Flow\Traits;
+
+/**
+ * Trait FlowIdentifierTrait
+ * @package Isobar\Flow\Traits
+ */
+trait FlowIdentifierTrait
```

Comment:
See above: This should be an interface not a trait, since it's acting as a contract not an implementation.